platform/platform-scheduler/src/main/resources/quartz.properties

```diff
@@ -4,18 +4,18 @@
 # Only difference from main file is the 
 # org.quartz.scheduler.skipUpdateCheck is set to true
 
-org.quartz.scheduler.instanceName: DefaultQuartzScheduler
-org.quartz.scheduler.rmi.export: false
-org.quartz.scheduler.rmi.proxy: false
-org.quartz.scheduler.wrapJobExecutionInUserTransaction: false
+org.quartz.scheduler.instanceName = DefaultQuartzScheduler
+org.quartz.scheduler.rmi.export = false
+org.quartz.scheduler.rmi.proxy = false
+org.quartz.scheduler.wrapJobExecutionInUserTransaction = false
 
-org.quartz.threadPool.class: org.quartz.simpl.SimpleThreadPool
-org.quartz.threadPool.threadCount: 10
-org.quartz.threadPool.threadPriority: 5
-org.quartz.threadPool.threadsInheritContextClassLoaderOfInitializingThread: true
+org.quartz.threadPool.class = org.quartz.simpl.SimpleThreadPool
+org.quartz.threadPool.threadCount = 10
+org.quartz.threadPool.threadPriority = 5
+org.quartz.threadPool.threadsInheritContextClassLoaderOfInitializingThread = true
 
-org.quartz.jobStore.misfireThreshold: 60000
+org.quartz.jobStore.misfireThreshold = 60000
 
-org.quartz.jobStore.class: org.quartz.simpl.RAMJobStore
+org.quartz.jobStore.class = org.quartz.simpl.RAMJobStore
```

Comment:
I updated this file to use the usual key/value delimiter in DDF. I tested updating properties in this file and redeploying after this update.